bid.proto

```diff
@@ -15,25 +15,23 @@ message Bid {
     JubJubCompressed commitment = 5;
     BlsScalar elegibility_ts = 6;
     BlsScalar expiration_ts = 7;
-    uint64 bid_tree_storage_index = 8;
+    fixed64 bid_tree_storage_index = 8;
 }
 
 // Used to Request the creation of a Bid
 message BidTransactionRequest {
     BlsScalar k = 1;
-    uint64 value = 2;
+    fixed64 value = 2;
```

Comment:
Why Fixed instead of Uint? Just to know, maybe i'm missing something.

Reply:
It's a tiny bit more efficient - I think it saves 2 bytes for encoding the number.